.ci.yaml

```diff
@@ -3087,6 +3063,27 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac_android entrypoint_dart_registrant
+    recipe: devicelab/devicelab_drone
+    bringup: true
+    timeout: 60
+    properties:
+      caches: >-
+        [
+          {"name":"gradle","path":"gradle"}
+        ]
+      dependencies: >-
+        []
```

Comment:
Nit: these are not necessary as the platform level includes them: https://github.com/flutter/flutter/blob/master/.ci.yaml

Reply:
done